Cargo.toml

```diff
@@ -9,6 +9,7 @@ members = [
   "test-macro",
 ]
 resolver = "2"
+rust-version = "1.75"
```

Comment:
Question: since we add MSRV here, do we need to also have it in all individual `Cargo.toml` files? Or would this get inherited automatically?

Reply:
It get's inherited by the crates of the workspace. I will then remove them from the individual crates. The reason to have it in individual crates would be in case you would want to override this version.